### PR TITLE
attempted assessment

### PR DESCRIPTION
**Pull request by romeoyutig**
# Submitting Week 6 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [X ] Coding Challenges
- [X ] Coding Concepts
- [X ] Interview Questions

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:

Had a blocker on code challenge question 1, wasn't able to get test to pass. Would appreciate feedback, left pseudo code, and thought process.

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -19,10 +20,33 @@ const people = [
...
+// return a string with the information provided in the test variable using string interpolation
+
+// const sentenceMaker = (arr) => {
+//   let firstPart = arr.map((obj => obj.name))
+//   let secondPart = arr.occupation
+//   return firstPart
+// }
+//   console.log(sentenceMaker(people))
+
+// Had a blocker here, I'm able to extract the obj, however it returns the array twice. I was planning on transforming each element into a string (.join)? , and then using .toUpperCase on index zero, then perhaps splicing the strings together, then using string interpolation and transform back into an array. also don't think i had the right arguments initially for my .map() function.
```
Comment (TsunamiCode5) [inline code-challenges.test.js:51]: So when I copy and paste this into my code editor and run it through terminal it is only returning the first and last names of the people in the array. I think that an approach I can walk you through could be mapping (like you have) through the array and then we can declare a variable to hold the word after using a .split method. After that we would be able to target the first letter of the first name giving it the upper case method and then joining them together. Repeating this for last name and then returning a string interpolation of the first variable declared (holding the first name) + the second variable declared (holding the last name) and the rest of the sentence.

**Review by TsunamiCode5 — CHANGES_REQUESTED** [review]
Let me know if the pseudocode helped enough to try this question again (:

File: code-challenges.test.js (hunk 1 above)
Comment (romeoyutig) [reply code-challenges.test.js:51]: pseudo code helped guide me in the right direction. thanks !

**Review by TsunamiCode5 — APPROVED** [review]
Romeo, GREAT work in this assessment. Way to finish strong and push through. A lot of times the prompts get confusing with the many different approaches, but with that pseudocode you really dug deep. Thank you for always giving these such great effort because here we are Week 6 and your progress as a developer is outstanding! You should be proud of yourself, because I am of you. This assessment comprehension level is advanced beginner. Keep pushing, Romeo! You are doing so great.

**Pull request merged**